storage/src/nodestore/persist.rs

```diff
@@ -456,7 +456,7 @@ impl NodeStore<Committed, FileBacked> {
 
                 // Decrement gauge for writes that have actually completed
                 if completed_writes > 0 {
-                    #[expect(clippy::cast_possible_truncation)]
+                    #[expect(clippy::cast_possible_truncation, clippy::cast_precision_loss)]
```

Comment:
not understanding the lint problems here. some targets are triggering it and some aren't.